Lib/pdb.py

```diff
@@ -493,6 +493,16 @@ def _cmdloop(self):
             except KeyboardInterrupt:
                 self.message('--KeyboardInterrupt--')
 
+    def _update_file_mtime(self):
```

Comment:
The function name seems wrong - it's not "updating" the mtime if it's already there. It only sets it if it's not there.

Also, the docstring of  `_validate_file_mtime` is no longer correct - it's not "since last time we saw it".

Reply:
I changed it to `init`, do you think it's better? Also changed the docstring a bit.

Reply:
maybe "save"?

Reply:
Is "save" better than "init"? Because I think save is similar to update. In this specific case, we only care for the entries that are not intialized.

Reply:
init sounds to me like you are initialising the actual file_mtime, rather than saving a copy of it. How about save_initial_file_mtime?

Reply:
Okay that makes sense. I thought about it and I don't think we need to put it in the command path now. We used to do that because we want the error message on every command. Now we can do it once in setup for all the frames and leave `onecmd` alone.